src/index.ts

```diff
@@ -194,6 +194,17 @@ function setupOpenMRS() {
             moduleName
           }
         )
+      },
+      {
+        id: "capture-patient-photo",
```

Comment:
This is an extension that renders a component that is specific to capturing the patient's photo, this component does more heavy lifting for the feature like providing the concept complex, obs date in desired backend format. The registration component only implements the POST operation. However I can work on a followup commit to make this component more generic. The [concept](https://github.com/openmrs/openmrs-esm-patient-chart-widgets/blob/f883d440a93e659f8d4d17f2a71d37bddf7dbac4/src/widgets/attachments/capture-patient-photo.component.tsx#L33) can be configured in the registration esm?

Reply:
Well, this is patient-chart-widgets here; I mean how often (besides registration) will we need to take the patient's photo? Once? Twice? More often we'll take *some* photo.

What we could do is to leave this special variant here, but also provide (and use in the special variant) a more generic version.

Reply:
>Well, this is patient-chart-widgets here; I mean how often (besides registration) will we need to take the patient's photo? Once? Twice? More often we'll take some photo.


The component that captures 'a photo' is existing and purely generic:- [CameraUpload](https://github.com/samuelmale/openmrs-esm-patient-chart-widgets/blob/MF-405/src/widgets/attachments/camera-upload.component.tsx#L13), you can see that `CapturePatientPhoto` component references it [here](https://github.com/samuelmale/openmrs-esm-patient-chart-widgets/blob/MF-405/src/widgets/attachments/capture-patient-photo.component.tsx#L54-L59) with a configuration suitable for the patient photo feature.

>What we could do is to leave this special variant here, but also provide (and use in the special variant) a more generic version.

Okay, I agree that we really need to improve these components in regards to re-usability.

Reply:
I think we have a disconnect here. `CameraUpload` is a component yes, but I refer to it / want / need it to be (usable as) an extension. Otherwise some microfrontend cannot leverage this.

Reply:
I get what you mean, so let me work on making the `CapturePatientPhoto` more generic :)

Reply:
Updated PR, also opened a followup PR on Reg. --> https://github.com/openmrs/openmrs-esm-patient-registration/pull/65

FYI: In my recent commit, I also improved on the styling 👇 
![2021-02-02 18-19-36 2021-02-02 18_21_21](https://user-images.githubusercontent.com/26084581/106626245-8cf8b800-6588-11eb-9b4d-bf7b773e7816.gif)

Reply:
Oooppss, let me scrap-off the config ie. `config.concepts.patientPhotoUuid`

Reply:
>Oooppss, let me scrap-off the config ie. config.concepts.patientPhotoUuid


Done ☺️